static/src/javascripts/bootstraps/commercial.js

```diff
@@ -75,6 +77,9 @@ define([
         ['cm-ready', function () {
             mediator.emit('page:commercial:ready');
             userTiming.mark('commercial end');
+            robust.catchErrorsAndLog('ga-user-timing-commercial-end', function () {
```

Comment:
we've got this logged in more detail on our dashboards, do we need GA measuring it too?